php-practice/section1.php

```diff
@@ -193,5 +247,15 @@
 // Does the element at index 'id' get overwritten? What does that tell you?
```

Comment:
@paulstatezny  the ID index does not get overwritten because it sees the Indexes of 'id' and 'ID' as separate indexes. I'm assuming that it doesn't ignore capitalization.